pkg/manager/daemon_adaptor.go

```diff
@@ -160,6 +165,17 @@ func (m *Manager) BuildDaemonCommand(d *daemon.Daemon, bin string, upgrade bool)
 			command.WithID(d.ID()))
 	}
 
+	prefetchMap := prefetch.GetPrefetchMap()
+	if imageReference != "" {
+		prefetchfiles, ok := prefetchMap[imageReference]
+		if !ok {
+			log.L.Warnf("image reference '%s' not found in prefetch files map", imageReference)
```

Comment:
I suppose we can't throw a warning if pefetch list does not exist since the user can never start nri-prefetch to transport the prefech list.

Reply:
fixed! removed the warning log